contact.html portfolio_home.html portfolio.html portfolio_style.css

karenschulz opened the pull request:
portfolio_home.html

I design a very simple site. created a nav bar with a label with my name, aligned to the left and two links aligned to the right using flex.
the body has a little profile picture and some text and two links at the bottom then is the footer with copy rights.

contact.html

another simple design, this time the nav bar fills both sides, but the body has different sizes. I resized the profile picture.

still working on portfolio.html which proves more difficult to tame.

karenschulz committed:
created html file and started coding adding head, nav, body, footer

karenschulz committed:
created file, coded and added head, nav, body and footer

karenschulz committed:
added nav colour pink, changed buttons into links

karenschulz committed:
added display nav_bar

karenschulz committed:
added changes in anchors colours,padding in nav, and changed properties of margins

karenschulz committed:
put anchors in separate div

karenschulz committed:
added profile image

karenschulz committed:
added padding to nav_bar, body and footer

karenschulz committed:
added folder with images

karenschulz committed:
added other links inside body

karenschulz committed:
tried to change the siz of the welcome

karenschulz committed:
renamed picture

karenschulz committed:
changed nav tag inside body tag

karenschulz committed:
added picture to images file

karenschulz committed:
added h1 and h2 to welcome

karenschulz committed:
added h1,h2, align text to footer, changed color to label or correct label color

karenschulz committed:
added comments in works, <p>, put nav inside body and re-arranged poctures

karenschulz committed:
changed size bottom links, aligned it to center, getting closer to final prototype

karenschulz committed:
renamed image

karenschulz committed:
added class links, added class portfolio_page but kept in comment. Repaired flex and now seems to work, but needs twitching

karenschulz committed:
created contact.html

karenschulz committed:
resized image, re-arranged elements

karenschulz committed:
rearrenged thigs, but not much progress

karenschulz committed:
added .image

Emily774 commented on portfolio_style.css:
```diff
@@ -51,39 +46,50 @@ h1, h2 {
+}
+
 footer {
 	text-align: center;
```
You might want to change your sizes to percentages to make sure they will work with the mobile site, also I am not sure if the no-wrap element will work with mobiles without seeing the end product. You might also want to remove the bits that you have commented out if you are not using them. Overall looks great though!

karenschulz replied:
Hi, thanks for your comments. Good points!
I will work on the percentages next I think

karenschulz replied:
Hi Louse. I did the percentage bit and erased my comments I am not using. Thanks. When I update it you might have to go over them again (I will need your approval)

Emily774 commented on portfolio.html:
```diff
@@ -6,21 +6,25 @@
 		<title>My Portfolio</title>
```
I didn't see a DOCTYPE but that might just be github.

karenschulz replied:
I think it was github, because it is in the original doc. Thanks.

Emily774 commented on portfolio.html:
```diff
@@ -6,21 +6,25 @@
-		<div class="welcome"><img class="mypicture" src="images/profile_pic.png" alt="my_profile_picture">
-			<section>
-			<div class="heading"><h2>Welcome to my Portfolio</h2></div>
-			<div class="portfolio1"><img class="first_work" src="images/green_back.png" alt="my_first_work"><p>I love to do illustrations, check my creations</p>
```
These div don't seem to have a </div>, also the </section> tag needs indenting

karenschulz replied:
I went over indentings and tidied them up. I also tidied that bit of coding.

Emily774 commented on portfolio.html:
```diff
@@ -6,21 +6,25 @@
+			<div class="portfolio1"><p><img class="first_work" src="images/green_back.png" alt="my_first_work"></p><p>I love to do illustrations, check my creations</p></div>
+			<div class="portfolio2"><img class="second_work" src="images/eating.png" alt="my_second_work"><p>I love html and CSS</p></div>
+			<div class="portfolio3"><img class="third_work" src="images/full-pouch.png" alt="my_first_work"><p>I've written two children's books</p></div>
+		<div><p><a href="#">Home</a></p>
```
there doesn't seem to be a closing div tag on line 25, I think that is supposed to be the </div> tag for line 18?

karenschulz replied:
I tidied this up, it should be now around line 56 in portfolio.html

Emily774 commented on portfolio.html:
```diff
@@ -6,21 +6,25 @@
-			<div class="portfolio1"><img class="first_work" src="images/green_back.png" alt="my_first_work"><p>I love to do illustrations, check my creations</p>
-			<div class="portfolio2"><img class="second_work" src="images/eating.png" alt="my_second_work"><p>I love html and CSS</p>
-			<div class="portfolio3"><img class="third_work" src="images/full-pouch.png" alt="my_first_work"><p>I've written two children's books</p>>
-		<div><a href="#">Home</a>
```
I think this <div> needs to be a </div>

karenschulz replied:
I made the links... I guess that is what you meant?

Emily774 commented on portfolio.html:
```diff
@@ -6,21 +6,25 @@
+		
 	</body>
 	
 	<footer>C Copyrights. All rights reserved. Karen Schulz, January 2021</footer>
```
I think the <footer> tags need to be within the body tag

karenschulz replied:
hi, do you mean the footer needs to be inside the body tag?

karenschulz replied:
I corrected that. Thanks!

Emily774 commented on portfolio.html:
```diff
@@ -6,21 +6,25 @@
 	<body>
+		
 		<nav class="nav_bar">
 			<div><label class="label_karen_schulz" for="nav_bar">Karen Schulz</label></div>
```
I think you need to have 'label for="NavBar" before having a class. Or else have the class element in a seperate div or p tag

karenschulz replied:
changed the label issue.

Emily774 reviewed (commented):
Overall looks great, just a few instances where I don't think you've remembered to close tags 👍

karenschulz committed:
added <article> tag, added <h1> to links

karenschulz committed:
added .portfolio1,2,3

karenschulz committed:
some changes I cant remember

karenschulz committed:
changes I cant remember

karenschulz committed:
added flexibility for phone navigation, added a: hover a:focus

karenschulz committed:
added form to contact.html

karenschulz committed:
added links at the bottom of the form, and made all links active now

karenschulz committed:
made all links active

karenschulz committed:
added more mobile features tu css such as font resizing

karenschulz committed:
repaired what didnt work, separated three images in a different div so the flex applies to only them, plus other few twicks

karenschulz committed:
added .portfolio_page so the flex applies only on three items

karenschulz commented:
I submitted Louises comments and corrected various issues. Tidied up the code, changed indentation, repaired strayed 
 things, closed opened tags, etc. I also added a form in contact.html, made the links available, added a mobile friendly version in css, added a hover and focus to my links, etc.

mtmgoliath commented on contact.html:
```diff
@@ -0,0 +1,73 @@
+			<p>Karen Schulz</p>
+			
+			<div>
+				<a href="portfolio_home.html">Home</a>
```
I assume that this "portfolio_home.html" is your home page?
If it is, then it should be named index.html as this is standard practice =]

karenschulz replied:
Yes, thanks Mike. I changed it to index.html as you suggested. You are right.

mtmgoliath commented on contact.html:
```diff
@@ -0,0 +1,73 @@
+		
+		<article>
+			<div class="contact"><img class="picture-of-me" src="images/half_body.png" alt="my_profile_picture">
+		<br>
```
shouldn't be using br. You should accomplish the styling using css instead.
so if you wanted more space above the p tag then you could add padding-top to it in css

karenschulz replied:
br erased

karenschulz replied:
and followed your advise of using div, and styled them in my CSS. Thanks

mtmgoliath commented on contact.html:
```diff
@@ -0,0 +1,73 @@
+			
+			<a href=mailto:sirk.schulz@gmail.com>Email Me</a>
+		
+		<br>
```
replace br with css styling

karenschulz replied:
done!

mtmgoliath commented on contact.html:
```diff
@@ -0,0 +1,73 @@
+		
+		<form>
+			
+			<p>
+				<label>Where did you hear about me?</label>
+			</p>
+			
+			<p>
+				<label for="friend"><h6>Friend</h6></label>
+				<input type="radio" name="my-radio" id="friend" value="friend" />
+			</p>
+			
+			<p>
+				<label for="githut"><h6>Github</h6></label>
+				<input type="radio" name="my-radio" id="github" value="ngithub" />
+			</p>
+			
+			<p>
+				<label for="my-checkbox">Would you like me to send you my portfolio?</label>
+				<input type="checkbox" name="my-checkbox" value="checkbox" />
+			</p>
+			
+			<p>
+				<label for="email"><h6>Please enter your email:</h6></label>
+				<input type="email"/>
+				<input type="submit" value="submit"/>
+			</p>
```
i am not sure if p tags should be using inside of a form. I believe you should use divs here instead as these are not paragraphs

karenschulz replied:
I asked Rayna and she said it is possible to use<p> inside a form. You cannot use <p> whit <h1> for example. So I kept the <p> tags

mtmgoliath commented on portfolio.html:
```diff
@@ -0,0 +1,72 @@
+			
+			<div class="portfolio1">
+				<p>
+					<img class="first_work" src="images/green_back.png" alt="my_first_work">
```
I dont think this image needs to nested in a p tag. just image would work i believe

karenschulz replied:
I erased the <p> tag and you are right, it was OK.

mtmgoliath commented on portfolio.html:
```diff
@@ -0,0 +1,72 @@
+				
+			<div class="portfolio3">
+				<p>
+					<img class="third_work" src="images/full-pouch.png" alt="my_third_work">
```
for the alt info here and above please give more literal description of what the picture shows =]

karenschulz replied:
I added a longer description to the alt

mtmgoliath commented on portfolio_home.html:
```diff
@@ -0,0 +1,45 @@
+		
+		<div class="welcome">
+			
+			<img class="mypicture" src="images/profile_pic.png" alt="my_profile_picture">
```
in alt describe your profile pic, hair style, accessories, mood etc

karenschulz replied:
oh... really? it's quite funny! But I did what you suggested

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+html {
+	font-family: "Fredericka the Great";
+	color: #5f6161;
+	font-size: 10pt;
```
better to use 10px here instead of pt

and then you can go on to use rem for rest of the sizing on the page

karenschulz replied:
Ok, you are right. Changed it

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+}
+
+h1 {
+	font-size: 50pt;
```
should use rem here. not sure how pt converts to px but if it was 50px here you would put 5rem instead.

karenschulz replied:
I changed all fonts to rem as you suggested. Good one!

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+	justify-content: space-between;
+	background-color: #e69ccb;
+	color: white;
+	height: 50px;
+	padding: 10px;
```
use rem (5rem for these 2)

karenschulz replied:
sorry, I am confused... can we talk about this? ;)
I thought that rem was for font size only...

karenschulz replied:
Ok, I talked to Rayna and she said rem is used for fonts and paddings are OK in px. Cheers

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+
+.welcome {	
+	background-color: #f7daed;
+	font-size: 30pt;
+	padding-bottom: 200px;
```
use rem:
3rem;
20rem;

karenschulz replied:
changed it to rem

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+
+.contact {
+	background-color: #f7daed;
+	font-size: 20pt;
+	padding-bottom: 200px;
+	padding-top: 100px;
```
use rem

karenschulz replied:
I changed the font to rem, but the padding I left it in px

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+
+.content {
+	background-color: #f7daed;
+	font-size: 10pt;
```
rem

karenschulz replied:
done

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+	background-color: #f7daed;
+	font-size: 10pt;
+	text-align: center;
+	padding-left: 50px;
+	padding-right: 50px;
+	padding-bottom: 100px;
```
5rem;
5rem;
10rem;

karenschulz replied:
left it.

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+
+.my_portfolio {
+	background-color: #f7daed;
+	font-size: 10px;
```
1rem

karenschulz replied:
done

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+	display: flex;
+    flex-direction: column;
+	flex-wrap: wrap;
+	padding-left: 50px;
+	padding-right: 50px;
+	padding-bottom: 100px;
```
5rem;
5rem;
10rem;

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+}
+
+.links {
+	font-size:20pt;
```
rem

karenschulz replied:
done

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+
+.portfolio_page {
+	background-color: #f7daed;
+	padding-bottom: 10px;
```
1rem

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+}
+
+article, footer {
+	padding-left: 150px;
+	padding-right: 150px;
```
15rem

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+}
+
+.picture-of-me {
+	height: 220px;
```
22rem

mtmgoliath commented on portfolio_style.css:
```diff
@@ -0,0 +1,155 @@
+	}
+
+a {
+		font-size: 1rem;
```
too much indentation here

karenschulz replied:
done.

mtmgoliath reviewed (changes requested):
great stuff.

few minor things: like using rem instead of pt or px

also your home page should be called index.html

Emily774 reviewed (approved):
Hi, I think you've implemented all the changes I wanted you to make so you just need to do the ones Mike suggested. Good work! It looks great!

karenschulz committed:
added divs to separate items and to style them in CSS

karenschulz committed:
added styling for the divs from contact. I also changed the font sizes to rem. added a font sizing in the mobile version

karenschulz committed:
changed name file from portfolio_home to index as suggested in my peer review.

karenschulz committed:
I think I added some padding, my altvdescriptions made them longer took some p tags out

karenschulz committed:
added copyright logo

karenschulz committed:
added copyright logo

karenschulz committed:
added copyright logo

karenschulz committed:
changed file name from portfolio_home.html to index.html

mtmgoliath commented on contact.html:
```diff
@@ -0,0 +1,72 @@
+		 s
+	</body>
+	
+	<footer>&copy Copyright. All rights reserved. Karen Schulz, January 2021</footer>
```
you need a : after the copy

karenschulz replied:
Yes. thanks for reminding me. I already changed it

karenschulz replied:
Sorry... got confused... I did the : after the copy and in my browser and it looks wrong, that is why I decided to take it out. Will check other options then, because this is a strange one... Can you see the Copy symbol at all?

mtmgoliath commented on index.html:
```diff
@@ -0,0 +1,45 @@
+	
+	</div>
+	
+	<footer>&copy Copyright. All rights reserved. Karen Schulz, January 2021</footer>
```
need the : after copy again

karenschulz replied:
it should be updated

mtmgoliath commented on portfolio.html:
```diff
@@ -0,0 +1,66 @@
+	<body>
+		
+		<nav class="nav_bar">
+				<p>Karen Schulz</p>
```
i would probably use a h tag here for your name

karenschulz replied:
hummm I will see... will play with it but if it doesnt look ok will leave.

karenschulz replied:
I think unless you see the actual page, it is difficult to assess

mtmgoliath commented on portfolio.html:
```diff
@@ -0,0 +1,66 @@
+		
+		</article>
+		
+		<footer>&copy Copyright. All rights reserved. Karen Schulz, January 2021</footer>
```
: after &copy

karenschulz replied:
OK... it works with semicolon.

karenschulz replied:
At least with me, it doesn't work with : 
👍

karenschulz replied:
Problem solved

mtmgoliath reviewed (approved):
great stuff, just need to update a couple of little things like : after the &copy in  each file

karenschulz committed:
changed copyright symbol, changed my name at the nav size <h3> and changed size links at nav bar

karenschulz committed:
changed copyright symbol, My Name at the navigation bar <h3> and size of links

karenschulz committed:
changed h1,h2 properties to font-weight normal, added .nav_links to manage size of the links in the nav bar, added properties of sizing in mobile version(.hello_stranger, .title)

karenschulz committed:
changed copyright symbol, nav bar details such as my name and size of links. repaired a mispelling on my_portfolio

mtmgoliath commented on contact.html:
```diff
@@ -0,0 +1,72 @@
+		 s
+	</body>
+	
+	<footer>&copy; Copyright. All rights reserved. Karen Schulz, January 2021</footer>
```
: not ; 
??

mtmgoliath replied:
had a look and works fine so ignore this

mtmgoliath commented on portfolio.html:
```diff
@@ -0,0 +1,66 @@
+		<div class="my_portfolio">
+				
+			<div class="portfolio2">
+					<img class="second_work" src="images/eating.png" alt="my second work hasmter eating a seed, fucsia background">
```
spelling mistake hamster

mtmgoliath replied:
also i believe it is fuchsia

mtmgoliath reviewed (approved):
only a couple of typos to be fixed and pushed up before you merge =]
